test/linalg/symmetric.jl

```diff
@@ -336,3 +336,13 @@ end
     @test ishermitian(Symmetric(B, :L))
     @test issymmetric(Hermitian(B, :L))
 end
+
+@testset "$HS solver with $RHS RHS - $T" for
+    HS  in (Hermitian, Symmetric),
```

Comment:
this looks like it's part of the body of the block, would be clearer to indent these line continuations of the block-opening line an extra level, and break at commas instead of between `for` and the variable